actionview/lib/action_view/template/handlers/erb.rb

```diff
@@ -58,6 +58,7 @@ def add_expr_escaped(src, code)
         def add_stmt(src, code)
           flush_newline_if_pending(src)
           super
+          flush_newline_if_comment(src, code)
```

Comment:
Instead of checking for a comment, would anything break if `add_stmt` simply always did `src << "\n"`?